### ISSUE TITLE
[ENHANCEMENT] PrusaConnect should report a filament runout.

### ISSUE THREAD

**Issue by derlucag** [prusa3d/Prusa-Firmware-Buddy#3145]
**Printer type** - MK4

**_TLDR: When filament runs out, and printer beeps and requests new filament, PrusaConnect should act accordingly._**

With much pain i migrated to PrusaConnect from Octoprint (which still works perfect with my MK3S+), since the MK4 does not play along well with Octoprint (just yet, i hope).

When some minutes ago my filament ran out, PrusaConnect just continued to report the Status "Printing". No notifications were sent to my Telegram Bot (which otherwise "works", meaning it say when prints start and stop).

Why even offer notifcations if one of the most basic and often reoccuring situation is not handled by one. 

It is infurating that PrusaConnect/Link does not have the most basic functionalities to monitor prints, like adding a custom camera stream without sacrificing a smartphone/tablet, or reporting the state of a filament runout sensor.

**Comment by cykedev** [comment]
+1

**Comment by Prusa-Support** [comment]
Thanks for your feedback.

The Prusa Connect platform is currently functioning and is open to the public, but please notice it is still displayed as _beta version_. We received several requests of this kind but being it "a beta", there is still work to be done.

We don't have a specific GitHub repository for Prusa Connect at the moment but most of Prusa Connect issues are listed in the [prusa3d/Prusa-Link-Web](https://github.com/prusa3d/Prusa-Link-Web) repository for the time being, so I'll move the issue there.

Michele Moramarco
Prusa Research
